roles/lib_openshift/src/lib/volume.py

```diff
@@ -0,0 +1,36 @@
+# pylint: skip-file
+
+class Volume(object):
```

Comment:
There is a stand alone for oc_volume.  It is on the list o things to port.  We are using its library calls under the covers.  Will port when I get a chance.